docs/version0.19.md

```diff
@@ -69,6 +70,24 @@ The `jpackage` utility is described in JEP 343 as a tool that "packages a Java a
 
 Linux x86 64-bit, Linux on POWER&reg; LE 64-bit, and Linux on IBM Z&reg; 64-bit have all moved to the gcc 7.5 compiler. See [Supported environments](openj9_support.md).
 
+### ![Start of content that applies to Java 14 plus](cr/java14plus.png) New JDK 14 features
+
+The following features are supported by OpenJ9:
+
+- [JEP 343](https://openjdk.org/jeps/343): Packaging Tool (Incubator)
```

Comment:
This should refer to the restrictions, https://github.com/eclipse-openj9/openj9-docs/issues/474